### PR TITLE
Storybook some stellar send form fields

### PR DESCRIPTION
**Pull request by zanderz**
Footer buttons and input warnings.
Note the zeppelin design shows a different style of input boxes (2 horizontal lines instead of a box) that I assume will be in a different PR?
image
image
image
image

### CODE REVIEW

File: shared/wallets/send-form/footer/index.js [hunk 1]
```diff
@@ -1,15 +1,55 @@
...
 }
 
-const Footer = ({onClick}: Props) => (
-  <Box2 direction="vertical">
-    <Button type="Primary" label="Send" onClick={onClick} />
+const Footer = ({onClickSend, onClickRequest, disabled}: Props) => (
+  <Box2 direction="vertical" fullWidth={true}>
+    <Box2
+      direction="horizontal"
+      style={{flex: 1, alignItems: 'stretch', justifyContent: 'center'}}
```
Comment (chrisnojima) [inline shared/wallets/send-form/footer/index.js]: please use stylesheetcreate for all new components

**Comment by zanderz** [comment]
@chrisnojima Changing the icon to a stylesheet style wouldn't pass flow for me.

**Comment by adamjspooner** [comment]
Did you use `iconCastPlatformStyles`?

**Comment by zanderz** [comment]
Hmm I thought I had tried that, but it's working now, thanks.

**Pull request merged**